Platform/ElkhartlakeBoardPkg/CfgData/CfgData_Tcc_Feature.dlt

```diff
@@ -0,0 +1,29 @@
+## @file
+#
+#  TCC feature Configuration Delta File.
+#
+#  Copyright (c) 2021, Intel Corporation. All rights reserved.<BR>
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+#
+
+# Enable TCC config data
+TCC_CFG_DATA.TccEnable   | 1
+TCC_CFG_DATA.TccTuning   | 1
+TCC_CFG_DATA.TccSoftSram | 1
+TCC_CFG_DATA.TccErrorLog | 1
+
+# Enable Timed GPIO
+SILICON_CFG_DATA.EnableTimedGpio0 | 1
+SILICON_CFG_DATA.EnableTimedGpio1 | 1
+
+# Enable RTCM from the boot option
+BOOT_OPTION_CFG_DATA_0.BootFlags_0 | 24

```

Comment:
It looks the first boot option will enable Yocto Mender(BIT4), PreOs(BIT2) and RTCM( BIT3) ,
so the value should be set to BIT4 + BIT2 + BIT3 =28

Reply:
for the first boot option, do you want to enable PreOS and TCC RTCM at same time? if yes, need set to 28 as comments.

Reply:
@gdong1  Thanks for catching this. I added without understanding what it is for. Done and added comment also for future understanding!